content/en/blog/argo-insurance-workflow/index.md

```diff
@@ -0,0 +1,125 @@
+postfirstname                        [GET]  /postfirstname          set-first-name    false   *    /postfirstname              
+postlastname                         [GET]  /postlastname           set-last-name     false   *    /postlastname               
+postsalary                           [GET]  /setmonthlysalary       set-salary        false   *    /setmonthlysalary  
+```
+
+### Running the application
+
+The above function helped us in creating independent logics. Now we are going to club them together and create a workflow using Argo. This workflow takes its decision on basis of input provided.
+
+```bash
+argo submit --watch insurance/insuranceDag.yaml 
+
+Name:                hello-argo-4n4pm
+Namespace:           fission
+ServiceAccount:      unset (will run with the default ServiceAccount)
+Status:              Running
+Created:             Tue Sep 20 10:56:57 +0530 (10 seconds ago)
+Started:             Tue Sep 20 10:56:57 +0530 (10 seconds ago)
+Duration:            10 seconds
+Progress:            0/4
+Parameters:          
+  age:               35
+  salary:            20000
+
+STEP                   TEMPLATE          PODNAME  DURATION  MESSAGE
+ ● hello-argo-4n4pm    insurance                              
+ ├─◷ getage            getage                                 
+ ├─◷ getmonthlysalary  getmonthlysalary                       
+ ├─◷ postfirstname     postfirstname                          
+ └─◷ postlastname      postlastname                           
+
+This workflow does not have security context set. You can run your workflow pods more securely by setting it.
+Learn more at https://argoproj.github.io/argo-workflows/workflow-pod-security-context/
+```
+
+`--watch` will help us to follow the status of workflow. You can view the logs/output of the Workflow using the below command,
+
+```bash
+$ argo logs -f hello-argo-4n4pm
+
+hello-argo-4n4pm-761537852:  _______________________ 
+hello-argo-4n4pm-761537852: < {Hello Neha Gupta!! as per your age: 35 We can give you an insurance with assured money 20000} >
+hello-argo-4n4pm-761537852:  ----------------------- 
+hello-argo-4n4pm-761537852:     \
+hello-argo-4n4pm-761537852:      \
+hello-argo-4n4pm-761537852:       \     
+hello-argo-4n4pm-761537852:                     ##        .            
+hello-argo-4n4pm-761537852:               ## ## ##       ==            
+hello-argo-4n4pm-761537852:            ## ## ## ##      ===            
+hello-argo-4n4pm-761537852:        /""""""""""""""""___/ ===        
+hello-argo-4n4pm-761537852:   ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~   
+hello-argo-4n4pm-761537852:        \______ o          __/            
+hello-argo-4n4pm-761537852:         \    \        __/             
+hello-argo-4n4pm-761537852:           \____\______/   
+hello-argo-4n4pm-761537852: time="2022-09-20T05:28:20.770Z" level=info msg="sub-process exited" argo=true error="<nil>"
+```
+
+`hello-argo-4n4pm` should be replaced by the Workflow name.
```

Comment:
Please add a conclusion/summary at the end of blogs.

You can see conclusion and want more section in the existing blogs.

Reply:
+1 need a conclusion with author bio link, you can refer to the last few blog posts.